src/modules/resetindex.js

```diff
@@ -0,0 +1,9 @@
+export default function resetIndex(objectListIndex) {
+  let sum = 0;
+  // eslint-disable-next-line no-restricted-syntax
+  for (const indexData of [...objectListIndex]) {
+    sum += 1;
+    indexData.index = sum;
+  }
```

Comment:
@IamYunusAli, I think if you refactor this:
`
sum += 1
`
to 

` 
sum = sum + 1
`
it should resolve the linter error.

Reply:
thank you for the feedback but the error still persist. its on the for/of loop
![reset](https://user-images.githubusercontent.com/32224137/177420655-f94d2972-f689-49ff-a134-c96580e81231.PNG)
.